website/docs/reference/resource-configs/access.md

```diff
@@ -33,7 +33,7 @@ There are multiple approaches to configuring access:
   ```
   </File>
   
-- In `properties.yml` using the new method (for v1.7 or higher). Use either the older method or the new method, but not both for the same model:
+- In `properties.yml` using the new method. Use either the older method or the new method, but not both for the same model:
```

Comment:
We can leave this one in since it's sort of a prerequisite
```suggestion
- In `properties.yml` using the new method (for v1.7 or higher). Use either the older method or the new method, but not both for the same model:
```